ide/gnatstudio/recordflux.py

```diff
@@ -128,6 +151,15 @@
       </menu>
    </submenu>
 
+    <!-- Context menu -->
+    <contextual action="rflx_display_graph">
+        <Title>RecordFlux/Display message graph</Title>
+    </contextual>
+
+    <contextual action="rflx_display_graph_unverified">
+        <Title>RecordFlux/Display message graph (unverified)</Title>
+    </contextual>
+
```

Comment:
Do we really want to have both? A new user will probably not understand the difference. As soon as one knows that the second option is much faster, only this variant will be used. I think it is okay to always skip verification for graph generation.

Reply:
I agree. Removed the second entry in 2e29cc1541353bc1394a62f90905dbf728547bda and always generate the graph without verification.